code/src/components/AddItemForm.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { todo } from 'reducers/todo'
+
+export const AddItemForm = () => {
+  const [name, setName] = useState('')
```

Comment:
maybe this should be called something else than name?
Like todo or task or text?